packages/components/select/src/select.tsx

```diff
@@ -34,7 +34,7 @@ interface SelectItemWithValue extends OptionProps {
 
 interface SelectItemWithItems extends OptionGroupProps {
   items?: SelectItemWithValue[]
-}
+} & UseSelectOptionGroupProps
```

Comment:
This fix is no longer needed in #2803. Please remove this modification.

Reply:
fixed